CodingTracker.DreamFXX/DatabaseManager.cs

```diff
@@ -0,0 +1,140 @@
+﻿using Spectre.Console;
+using Microsoft.Data.Sqlite;
+using Dapper;
+using CodingTracker.DreamFXX.Models;
+
+
+namespace CodingTracker.DreamFXX
+{
+    public class DatabaseManager
+    {
+        private readonly string? _connectionString = System.Configuration.ConfigurationManager.ConnectionStrings["DefaultConnection"].ConnectionString;
+
+        public void CreateDatabase()
+        {
+            using (var connection = new SqliteConnection(_connectionString))
+            {
+                connection.Open();
+                connection.Execute(@"CREATE TABLE IF NOT EXISTS MyCodingTracker (
+                                        Id INTEGER PRIMARY KEY AUTOINCREMENT,
+                                        Date TEXT,
+                                        StartTime TEXT,
+                                        EndTime TEXT,
+                                        Duration TEXT)");
+            }
+        }
+
+        public void InsertRecord(string? date, string? startTime, string? endTime, string duration)
+        {
+            using (var connection = new SqliteConnection(_connectionString))
+            {
+                connection.Open();
+                connection.Execute(@$"INSERT INTO MyCodingTracker (Date, StartTime, EndTime, Duration)
+                                       VALUES('{date}', '{startTime}', '{endTime}', '{duration}')");
+            }
+        }
+
+        public void DeleteRecord(string? date)
+        {
+            using (var connection = new SqliteConnection(_connectionString))
+            {
+                connection.Open();
+                string sql = @$"DELETE FROM MyCodingTracker
+                                            WHERE Date = '{date}'";
+                connection.Execute(sql);
+            }
+        }
+
+        public void UpdateRecord(string? oldDate, string? newDate, string? startTime, string? endTime, string? duration)
+        {
+            using (var connection = new SqliteConnection(_connectionString))
+            {
+                connection.Open();
+                
+                    if (startTime is null && endTime is null)
+                    {
+                        connection.Execute($@"UPDATE MyCodingTracker
```

Comment:
🔴 **SQL Injection**

Can you change to use parameters
https://www.learndapper.com/saving-data/insert